SPBypassLoginPage/LOGIN/Bypass/BypassLogin.aspx.cs

```diff
@@ -85,11 +91,12 @@ protected override void OnLoad(EventArgs e)
                 SPHttpUtility.NoEncode((string)HttpContext.GetGlobalResourceObject("wss", "login_pagetitle", System.Threading.Thread.CurrentThread.CurrentUICulture));
             ClaimsLogonPageMessage.Text = SPHttpUtility.NoEncode(SPResource.GetString(Strings.SelectAuthenticationMethod));
 
+            var loginMode = LoginMode;
```

Comment:
This change looks unnecessary, can you remove local var loginMode and use directly LoginMode?

Reply:
I've made that change for performance reasons. This code gets executed on every single request - so it should as performant as possible... The implementation of LoginMode property loads the farm property values on every call...

Reply:
Ok, then instead of creating the var in the method, I introduced private member m_LoginMode to cache the result. 
But the downside is that if you change the setting, the only way to get it applied by the page is to recycle the application pool.
I still didn't make my mind if the performance gain (very little IMO) is worth this downside.
What do you think?

Reply:
No, it doesn't require an application pool recycle - that value gets cached on a per-request basis. I only wanted to ensure that the same code doesn't get executed twice...

Reply:
Indeed, I get rusty with WebForms...